### DISCUSSION TITLE
can webui split frontend from backend if I need to

### DISCUSSION THREAD

**Discussion by laoshaw** [webui-dev/webui#458] [General]
webui is for a GUI running on local machine using websocket for local http server as the backend.
can I also make that backend running on a remote server, so it can be running either as a single GUI app on the local machine, or a client-server combo when needed?

**Comment by hassandraga** [comment]
Yes, you can check how to by looking at the public access example. And the multi clients example.

Basically, use start server only API, set it to public, and allow multiple clients.

**Comment by laoshaw** [comment]
that's great, thanks!
